elogia_hr/report/hr_leave_attendance_report_calendar.py

```diff
@@ -1,11 +1,62 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
-from datetime import date
-
 from odoo import api, fields, models, tools, SUPERUSER_ID, _
-
 from odoo.addons.base.models.res_partner import _tz_get
 
+from datetime import date, datetime
+from dateutil.relativedelta import relativedelta
+
+MONTHS = ("Enero",
+          "Febrero",
+          "Marzo",
+          "Abril",
+          "Mayo",
+          "Junio",
+          "Julio",
+          "Agosto",
+          "Septiembre",
+          "Octubre",
+          "Noviembre",
+          "Diciembre")
+
+
+class LeaveAttendanceReport(models.Model):
+    _name = "hr.leave.attendance.report"
+    _description = 'Time Off Attendance Report'
+
+    name = fields.Char(string='Name', index=True, required=True)
+    employee_id = fields.Many2one('hr.employee', string='Employee')
+    start_datetime = fields.Date(string='From')
+    stop_datetime = fields.Date(string='To')
+    on_holiday = fields.Boolean('On holiday')
+    on_attendance = fields.Boolean('On attendance')
+    actual_year = fields.Integer('Actual Year')
+    actual_month = fields.Integer('Actual Month')
+    actual_day = fields.Integer('Actual Day')
+
+    @api.model
+    def update_calendar_by_year(self):
+        init_date = date.today()
+        finish_date = datetime.strptime('31-12-' + str(date.today().year), '%d-%m-%Y').date()
+        date_list = [init_date + relativedelta(days=d) for d in range((finish_date - init_date).days + 1)]
+        obj_employee_ids = self.env['hr.employee'].search([('active', '=', True)])
```

Comment:
not sure si va igual para hr_employee, pero el campo active en odoo por defecto no te da solo los True??

Reply:
Lo acabo de probar, he archivado un empleado y al hacer la consulta active=True, me trajo todo menos los achivados.

Reply:
me refiero a que si tu dices self.env['hr.employee'].search([]), solo deberia traer los que estan activos, sin necesidad de especificar active=True

Reply:
Listo!!